ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -90,6 +91,7 @@ descriptor:
   inlet: "Area of media entrance."
   input: "The input to a system."
   isolation: "Process of isolating one component or process from another."
+  lamp: "Type of light dimming command point."  # Added 25/08/20
```

Comment:
This definition is not useful (its the same definition as was used for group, and is uninformative regarding what a 'lamp' is).